bedrock/base/templates/includes/protocol/navigation/index.html

```diff
@@ -25,9 +25,7 @@
         <div class="mzp-c-navigation-download">
           {{ download_firefox(alt_copy=_('Download Firefox'), dom_id='protocol-nav-download-firefox', button_color='mzp-t-secondary mzp-t-small', download_location='nav') }}
           <div class="c-navigation-fxa-cta-container">
-            <a class="c-navigation-fxa-cta mzp-c-button mzp-t-secondary mzp-t-small mzp-t-product" {{ fxa_link_fragment(entrypoint='mozilla.org-globalnav', action='signup', utm_params={'campaign': 'globalnav', 'content': 'get-firefox-account', 'medium': 'referral', 'source': 'www.mozilla.org'}) }} data-button-name="Get a Firefox Account" data-link-type="FxA-Sync" data-cta-position="secondary cta" data-alt-href="{{ url('firefox.accounts') }}">
-              {{ _('Get a Firefox Account') }}
-            </a>
+            {{ fxa_cta_link(entrypoint='mozilla.org-globalnav', action='signup', button_text=_('Get a Firefox Account'), button_class='c-navigation-fxa-cta mzp-c-button mzp-t-secondary mzp-t-small mzp-t-product', button_location='secondary cta', content='get-firefox-account', campaign='globalnav', source='www.mozilla.org') }}<br>
```

Comment:
data-alt-href="{{ url('firefox.accounts') }}" is missing here, which is breaking this logic https://github.com/mozilla/bedrock/blob/master/media/js/base/protocol/init-navigation.js#L59-L70

It looks like we're also missing an `account_id` here

Reply:
Only the account benefits page seemed to have an id defined, which was styles related. Is the way I wrote the macro mean that `account_id` is not optional?

Reply:
Looking in the generated HTML, the button currently outputs an empty `id=""` if an Id is not passed. Maybe just add it as an optional parameter?

Reply:
Now that I'm thinking of it, maybe this particular one doesn't need to be a `fxa_cta_link` macro. It doesn't make sense to at data-alt-href to the macro because this feels like the only place that uses it that I've come across. It may make more sense to just add the class of `js-fxa-cta-link`?